### PR TITLE
Feat(eos_config_deploy_cvp): Added tag containers for Building Containers topology task

### LINKED ISSUE #1181: Add tags for tasks at eos_config_deploy_cvp to achieve "configlet update" only
### Enhancement summary

Currently both "Create Configlets" and "Build Container and Topology" plays in present.yml use same single "Apply" tag.
In order to run AVD push for network services only update (no change to containers/nodes) suggestion is to add extra tag "Topology" for "Build Container and Topology" play, so one can use --skip-tags option to skip this plays during playbook execution

### Which component of AVD is impacted

eos_config_deploy_cvp

### Use case example

Adding servers/network services in AVD is more frequent task in Day2 operations. If network fabric is not changed, no need to check container structure in CVP during AVD CVP push. This tag will allow to skip it if operator is sure that it is not needed.

### Describe the solution you would like

suggestion is to add extra tag "Topology" for "Build Container and Topology" play, so one can use --skip-tags option to skip this plays during playbook execution

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by lermilov**
## Change Summary
Adding extra tag "containers" for "Build Container and Topology" play, so one can use --skip-tags option to skip this plays during playbook execution

## Related Issue(s)

Fixes #1181

## Component(s) name

`arista.avd.eos_config_deploy_cvp`

## Proposed changes
Adding extra tag "containers" for "Build Container and Topology" play, so one can use --skip-tags option to skip this plays during playbook execution.

## How to test
ansible-playbook push-to-cvp.yml --skip-tags containers

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by titom73 — COMMENTED** [review]
Useful PR, thanks. Could you also update README file of the role to add this new tag in the documentation ?

**Review by titom73 — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/roles/eos_config_deploy_cvp/README.md [hunk 1]
```diff
@@ -211,6 +211,15 @@ This module also supports tags to run a subset of ansible tasks:
...
+Other option to run a subset of ansible tasks is to use __`--skip-tags <tag>`__:
+- in order to run module to update existing configlets only, following command can be used:
+```shell
+$ ansible-playbook playbook.to.deploy.with.cvp.yml --skip-tags "containers"
+```
+- Skipping multiple tags could make playbook even more lightweight. For example, above command with avoiding CVP task execution
+```shell
+$ ansible-playbook playbook.to.deploy.with.cvp.yml --skip-tags "containers,apply"
+```
 ### Outputs
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_config_deploy_cvp/README.md:224]: missing empty line before next heading

**Review by carlbuchmann — APPROVED** [review]
LGTM!

**Pull request merged**